securedrop/models.py

```diff
@@ -302,7 +286,7 @@ def __init__(self,
     def __repr__(self) -> str:
         return '<Reply %r>' % (self.filename)
 
-    def to_json(self) -> "Dict[str, Union[str, int, bool]]":
+    def to_json(self) -> 'Dict[str, Any]':
```

Comment:
Why relax this?

Reply:
It was causing mypy errors, and `Dict[str, any]` is, for now, the "official" way to define JSON data (https://github.com/python/typing/issues/182).

Reply:
This is because each value could be a dictionary of other values.